### PR TITLE
Change the license from MIT to Apache 2.0

### PR DESCRIPTION
**Pull request by utam0k**
Not that there was anything wrong with it, but I set the license first without thinking about it, so I'll change it to Apache 2.0, which is more explicit.

### CODE REVIEW

**Comment by utam0k** [comment]
@Furisto @yihuaf @nimrodshn @tsturzl @YJDoc2 
I'll merge this PR after I get approval from all collaborators.

**Review by Furisto — APPROVED** [review]

**Review by tsturzl — APPROVED** [review]

**Review by yihuaf — APPROVED** [review]

**Review by YJDoc2 — APPROVED** [review]

**Comment by YJDoc2** [comment]
Hey , should we be adding a contributing notice, along with a copyright notice as applicable? As far as I know apache license is more concrete in terms of copyright ,  so maybe we should add appropriate notice in readme if required?

**Review by nimrodshn — APPROVED** [review]

**Comment by utam0k** [comment]
> Hey , should we be adding a contributing notice, along with a copyright notice as applicable? As far as I know apache license is more concrete in terms of copyright , so maybe we should add appropriate notice in readme if required?

Thank you for your input. Do you have any good examples?

**Pull request merged**
